src/prometheus.rs

```diff
@@ -0,0 +1,36 @@
+pub use election_status::record_election;
+pub use metrics_exporter_prometheus::PrometheusHandle;
+
+use metrics::describe_gauge;
+use metrics_exporter_prometheus::PrometheusBuilder;
+
+pub fn setup_metrics_recorder() -> anyhow::Result<PrometheusHandle> {
+    let handle = PrometheusBuilder::new().install_recorder()?;
+    describe_gauge!(election_status::TARGET, election_status::DESCRIPTION);
+    metrics::gauge!(election_status::TARGET)
```

Comment:
I had in mind to trigger alarms for the following:

1. Election failed - fatal alarm
2. Election was completed by unsigned solution (offchain worker)

I would say that they are used for slightly different use-cases, in this scenario we only want to know whether the most recent election i.e, success (signed), success (unsigned) or failed.  

With a Counter/CounterVec we can just increment or decrement the value with a certain label which isn't really suitable to trigger alarms for the most recent elections. Because once you a failure it's not possible to know whether the error happen "now" or "in the past" unless the caller store the count somewhere. 

I would say they are doing different things and I think both would be useful but could be done in another PR or query via the web APIs :)